examples/kubernetes/cluster/main-datasource.tf

```diff
@@ -0,0 +1,6 @@
+data "vkcs_kubernetes_cluster" "k8s-cluster" {
+  cluster_id = vkcs_kubernetes_cluster.k8s-cluster.id
+  # This is required here to let the example work with just created cluster.
+  # In Real life u wont have to wait for it to be created 
```

Comment:
"u"??? Just copy the same text from other examples

Reply:
removed it if we do not need to link by depends on